Lib/ufo2ft/featureWriters/cursFeatureWriter.py

```diff
@@ -19,6 +19,8 @@ class CursFeatureWriter(BaseFeatureWriter):
 
     tableTag = "GPOS"
     features = frozenset(["curs"])
+    entry = "entry"
```

Comment:
maybe better to have an `options` dict so you don't need to subclass to modify entry/exit anchor names, you simply pass the desired names to the default constructor as keyword arguments:

```python
class CursFeatureWriter(BaseFeatureWriter):
    ...
    options = dict(entry="entry", exit="exit")    
```

Then you make an instance of CursFeatureWriter(entry="foobar", exit="baz")

KernFeatureWriter has a similar thing for `ignoreMarks`: 
https://github.com/googlefonts/ufo2ft/blob/236096cc9699649b965b24459f85b0fb6bd97264/Lib/ufo2ft/featureWriters/kernFeatureWriter.py#L110

Reply:
The parent BaseFeatureWriters automatically turns that `options` dict attribute of the class into a self.options namespace, see
https://github.com/googlefonts/ufo2ft/blob/236096cc9699649b965b24459f85b0fb6bd97264/Lib/ufo2ft/featureWriters/baseFeatureWriter.py#L59-L64

Reply:
an additional advantange of using `options` dict, is that the custom entry/exit anchor names can be encoded as options directly in the featureWriters lib key.

```xml
<dict>
    <key>com.github.googlei18n.ufo2ft.featureWriters</key>
    <array>
        <dict>
            <key>class</key>
            <string>CursFeatureWriter</string>
            <key>options</key>
            <dict>
                <key>entry</key>
                <string>foobar</string>
                <key>exit</key>
                <string>baz</string>
            </dict>
        </dict>
    </array>
</dict>
```